extensions/parquet/table/src/test/java/io/deephaven/parquet/table/ParquetTableReadWriteTest.java

```diff
@@ -1204,7 +1231,8 @@ public void partitionedParquetWithDotFilesTest() throws IOException {
         writeTable(someTable, firstDataFile);
         writeTable(someTable, secondDataFile);
 
-        Table partitionedTable = readKeyValuePartitionedTable(parentDir, EMPTY).select();
+        final URI parentURI = convertFileToURI(parentDir, true);
+        final Table partitionedTable = readTable(parentURI.toString()).select();
```

Comment:
Why the test change from `readKeyValuePartitionedTable` to `readTable`?

Reply:
I was testing if we pass a string source, we can still read a key value partitioned table